### PR TITLE
Added "go up" link to browse template

### PR DESCRIPTION
**Pull request by tboerger**
In order to have directly a link within the browse listing I have added
a link to the top of the table to get one level up in the tree. Added
that after a chat with @mholt.

Signed-off-by: Thomas Boerger <thomas@webhippie.de>

### CODE REVIEW

**Comment by tboerger** [comment]
bildschirmfoto 2016-04-11 um 19 33 50

File: caddy/setup/browse.go [hunk 1]
```diff
@@ -342,6 +343,17 @@ footer {
...
+					{{if .CanGoUp}}
+						<tr>
+							<td>
+								<a href="..">
+									<span class="goup">Go up</span>
+								</a>
+							</td>
+							<td>&mdash;</td>
+							<td>&mdash;</td>
+						</tr>
```
Comment (mholt) [inline caddy/setup/browse.go]: Nit-pic: Un-intent this whole `<tr>...</tr>` block 1 notch so it is consistent with the other ones.

**Comment by mholt** [comment]
Looking visually good! :+1:

**Comment by tboerger** [comment]
@mholt applied your comment

**Comment by mholt** [comment]
Thank you Thomas!

**Pull request merged**